qemu/tests/qmp_command.py

```diff
@@ -262,10 +245,6 @@ def send_cmd(cmd):
     cmd_return_value = params.get("cmd_return_value")
     exception_list = params.get("exception_list", "")
 
-    # HOOKs
-    if result_check == 'qmp_cpu':
-        pre_cmd = "cpu index=%d" % (int(params['smp']) - 1)
-
```

Comment:
For the reason of deleted, I can add some explain message 
For the function, I am not sure we can use the result_check function, because the first parameter of function is used for "param qmp_o: output from pre_cmd, qmp_cmd or post_cmd.", I think it doesn't make sense when used with "len(output)" :)  Many thanks